### PR TITLE
docs: reorganise example notebooks

### PR DESCRIPTION
**Pull request by redeboer**
- An additional test has been added to see if the notebooks execute properly
- For this, the notebooks had to be rewritten
- The three examples have been put into different pages

### CODE REVIEW

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

File: pycompwa/plotting/__init__.py [hunk 1]
```diff
@@ -313,9 +316,11 @@ def make_dalitz_plots(plot_data, var_names, **kwargs):
         invariant_mass_names = var_names
     else:
         invariant_mass_names = [x for x in list(plot_data.data.dtype.names)
-                                if 'mSq' in x and '_vs_' in x]
+                                if 'mSq' in x
+                                and '_(' in x
+                                and x.endswith(')')]
 
-    data_weights = (plot_data.data.weight)
+    data_weights = (plot_data.data.weights)
```
Comment (spflueger) [inline pycompwa/plotting/__init__.py]: why this changed?

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)

File: doc/source/_pycompwa/pycompwa.rst [hunk 2]
```diff
@@ -1,19 +0,0 @@
-pycompwa package
```
Comment (spflueger) [inline doc/source/_pycompwa/pycompwa.rst]: why did you delete these two files?

Comment (redeboer) [reply doc/source/_pycompwa/pycompwa.rst]: Ah yeah, I thought at first they are just generated by `sphinx-apidoc` (which they are), but that would leave out the `ui` pybindings module.

**Comment by spflueger** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ComPWA/pycompwa/pullRequest?prid=5168854&bid=16968273)
